src/sections/Blog-sidebar/index.js

```diff
@@ -6,29 +6,48 @@ import { FaSearch } from "react-icons/fa";
 
 import Button from "../../reusecore/Button";
 
-import WdThumb1 from "../../assets/images/blog/widgets-thumb/01.png";
-import WdThumb2 from "../../assets/images/blog/widgets-thumb/02.png";
-import WdThumb3 from "../../assets/images/blog/widgets-thumb/03.png";
 
 import BlogSideBarWrapper from "./blogSidebar.style";
 
 const Sidebar = ( ) => {
     const data = useStaticQuery(
         graphql`
-            query allTags {
-                allMdx(
+            query allTagsAndLatestPosts {
+                allTags: allMdx(
                     filter: { fields: { collection: { eq: "blog" } }, frontmatter: { published: { eq: true } } }
                 ){
                     group(field: frontmatter___tags) {
                         fieldValue
                         totalCount
                     }
                 }
+
+                latestPosts: allMdx(
+                    sort: { fields: [frontmatter___date], order: DESC}
+                    filter: { fields: { collection: { eq: "blog" } }, frontmatter: { published: { eq: true } } }
+                    limit: 4      
+                ) {
+                    nodes {
+                        frontmatter {
+                            title
+                            date(formatString: "Do MMMM YYYY")
+                            author
+                            thumbnail{
```

Comment:
Can you use this like in other it has been used in other queries i.e also using `childImageSharp`. Then `Image` component should be used below instead of `img`

Reply:
> Can you use this like in other it has been used in other queries i.e also using `childImageSharp`. Then `Image` component should be used below instead of `img`

Ya, I'll do that too